tests/helpers/module-for-acceptance.js

```diff
@@ -1,23 +1,23 @@
 import { module } from 'qunit';
+import Ember from 'ember';
 import startApp from '../helpers/start-app';
 import destroyApp from '../helpers/destroy-app';
 
+const { RSVP: { Promise } } = Ember;
+
 export default function(name, options = {}) {
   module(name, {
     beforeEach() {
       this.application = startApp();
 
       if (options.beforeEach) {
-        options.beforeEach.apply(this, arguments);
+        return options.beforeEach.apply(this, arguments);
       }
     },
 
     afterEach() {
-      if (options.afterEach) {
-        options.afterEach.apply(this, arguments);
-      }
-
-      destroyApp(this.application);
+      let afterEach = options.afterEach && options.afterEach.apply(this, arguments);
+      return Promise.resolve(afterEach).then(() => destroyApp(this.application));
```

Comment:
Ember CLI changed this file. I ran `ember init` and pulled in all the necessary changes.